admin/customers.php

```diff
@@ -1399,11 +1399,16 @@ function init() {
                     if ($customers_orders->RecordCount() != 0) {
 
                       $lifetime_value = 0;
+                      $last_order = array(
+                        'date_purchased' => $customers_orders->fields['date_purchased'],
+                        'order_total' => $customers_orders->fields['order_total'], 
+                        'currency' => $customers_orders->fields['currency'], 
+                        'currency_value' => $customers_orders->fields['currency_value']);
```

Comment:
Our standard formatting here is to end the array with a comma and new line, putting the closing bracket on the new line. This makes it easier to read and makes future diffs easier to parse since adding another element to the array wouldn't change any existing rows just to add the required comma.

Reply:
Noted.  Thanks.